src/validation/fields/address.validation.ts

```diff
@@ -3,14 +3,45 @@ import { defaultRequiredErrorMessages, defaultOptionalErrorMessages, ErrorMessag
 
 import {
   addressFieldsHaveNoValue,
+  checkPropertyNameOrNumberIfRadioButtonSelected,
   checkFieldIfRadioButtonSelected,
   checkFieldIfRadioButtonSelectedAndFieldsEmpty,
   checkInvalidCharactersIfRadioButtonSelected,
-  checkMaxFieldIfRadioButtonSelected
+  checkMaxFieldIfRadioButtonSelected,
+  checkPresenceOfPropertyNameOrNumber
 } from "../custom.validation";
 import { ErrorMessages } from "../error.messages";
 import { VALID_CHARACTERS } from "../regex/regex.validation";
 
+export const overseas_entity_address_validations = (errors: ErrorMessagesOptional = defaultOptionalErrorMessages) => {
+  errors = { ...defaultOptionalErrorMessages, ...errors };
+  return [
+    body("principal_address_property_name_number")
```

Comment:
seeing as this returns an a list of validators, some might be shared with the other lists, could we have a common list and use ... to import the common validations into the list mixed with the specific validations for that field/page, or wold that mean refactoring all the address validations?

Reply:
Don't know, but a sensible suggestion. We should investigate if this becomes a 'real' changes.